distribution/openrct2.d.ts

```diff
@@ -2781,6 +2781,14 @@ declare global {
          */
         lostCountdown: number;
 
+        /**
+         * The guest's current ride or stall destination id. Returns -1 if the guest is not headed to a ride or stall.
+         *
+         * If set to a value, guest will be sent to that ride.
+         * Has no effect on guests who are on a ride or outside of the park.
+         */
+        rideHeadedTo: number;
```

Comment:
I'd say `null` would be better to use instead of `-1`, and more consistent with the rest of the API.

Reply:
I think this is the right way to do it. Unfortunately I attempted to make this fix but couldn't get it working. 

I am able to change to return a ride instead of a rideID, which will then return null if there isn't a value set. The downside is then the getter and setter would have different signatures - the setter takes a ride id, and the getter returns a ride. 

This led to me initial split implementation the getter prop and the setter method, but @KatieZeldaKat suggested combining into one. I'm not sure how to move forward with this since I can't figure out how to implement it. I've pushed a change that returns a ride/null instead of a number/-1, but I'm not satisfied with it.

Reply:
What if it was `rideHeadedTo: number | null`, and if you set it to `null`, then a guest would no longer be heading to a ride. The get and set would match, and add functionality to stop a guest from heading somewhere.

Reply:
@ltsSmitty I meant `rideHeadedTo: number | null`, just like [`Car.nextCarOnTrain`](https://github.com/OpenRCT2/OpenRCT2/blob/409da62fe8e61dc9a64f9a6d8462ee5029d9fad9/distribution/openrct2.d.ts#L2459) as implemented [here](https://github.com/OpenRCT2/OpenRCT2/blob/409da62fe8e61dc9a64f9a6d8462ee5029d9fad9/src/openrct2/scripting/bindings/entity/ScVehicle.cpp#L171-L199). You don't need to return a `Ride`.

Reply:
@Basssiiie, that was exactly the reference I needed to make this happen. It's what I wanted all along, but I was struggling with DukValue.

I've implemented this and am really happy with the results.